### DISCUSSION TITLE
Extra breakpoints based on height.

### DISCUSSION THREAD

**Discussion by cristianjovanovich** [chakra-ui/chakra-ui#5229] [General]
First of all, I apologize for my English. I wanted to know if there is a way to create custom breakpoints, but based on the height of the screen.
Being able to do so:
const variant = useBreakpointValue({ baseHeight: 'outline', mdHeight: 'solid' })

**Comment by user72356** [comment]
I'm looking to do something like this as well!

**Comment by user72356** [comment]
Did you figure it out? I'm planning on looking at the useBreakpointValue code and creating a useHeightBreakpointValue using a separate breakpoint array from the theme.

**Comment by natromankevich** [comment]
This would be a nice feature!
